src/script/clientscripting.cpp

```diff
@@ -68,4 +72,11 @@ void ClientScripting::InitializeModApi(lua_State *L, int top)
 	LuaItemStack::Register(L);
 	StorageRef::Register(L);
 	LuaMinimap::Register(L);
+	LuaLocalPlayer::Register(L);
+}
+
+void ClientScripting::on_client_ready(LocalPlayer *localplayer)
```

Comment:
When you add documentation you should add a warning that the localplayer reference isn't always accessible.